std_msgs/Header header
int64 est_water_depth      # est_water_depth reading


